generators/entity-client/templates/react/src/main/webapp/app/entities/entity-detail.tsx.ejs

```diff
@@ -150,7 +151,7 @@ export const <%= entityReactName %>Detail = (props: I<%= entityReactName %>Detai
     {
         (<%= entityInstance %>Entity.<%= relationshipFieldNamePlural %>) ?
             (<%= entityInstance %>Entity.<%= relationshipFieldNamePlural %>.map((val, i) =>
-                <span key={val.id}><a>{val.<%= otherEntityField %>}</a>{(<%= entityInstance %>Entity.<%= relationshipFieldNamePlural %> && i === <%= entityInstance %>Entity.<%= relationshipFieldNamePlural %>.length - 1) ? '' : ', '}</span>
+                <span key={val.<%= otherEntityPkName %>}><a>{val.<%= otherEntityField %>}</a>{(<%= entityInstance %>Entity.<%= relationshipFieldNamePlural %> && i === <%= entityInstance %>Entity.<%= relationshipFieldNamePlural %>.length - 1) ? '' : ', '}</span>
```

Comment:
@gzsombor there is no primaryKey for embedded entities.
See https://github.com/hipster-labs/jhipster-daily-builds/runs/1843126537?check_suite_focus=true

Reply:
Interesting, I will have a look on it!